app/models/grda_warehouse/confidence/days_homeless.rb

```diff
@@ -20,20 +20,21 @@ def self.queue_batch force_run: false, force_create: false
       notifier = self.new.notifier
       message = "Generating confidence for days homeless"
       Rails.logger.info message
-      notifier.ping message if notifier
+      notifier&.ping message
```

Comment:
Surprised RuboCop didn't complain about the lack of parens...

Reply:
It only complains on `def`